src/care/KeyValueSorter.h

```diff
@@ -1206,7 +1206,7 @@ class KeyValueSorter<T, RAJA::seq_exec> {
       /// @return whether keys are allocated
       ///////////////////////////////////////////////////////////////////////////
       bool keysAllocated() const {
-         return m_keys ;
+         return m_keys != nullptr;
```

Comment:
This is the pattern that is fine to change. I'm fine with either what you have here or `return (bool) m_keys`. What you have here is probably actually better (easier to see what is going on).

Reply:
I kept it as != nullptr, sounds good